node/packages/aws-lambda-otel-extension/external/otel-extension-external/report-otel-data.js

```diff
@@ -18,42 +18,27 @@ const S3_BUCKET = process.env.SLS_OTEL_REPORT_S3_BUCKET;
 const LOGS_URL = process.env.SLS_OTEL_REPORT_LOGS_URL;
 const REQUEST_RESPONSE_URL = process.env.SLS_OTEL_REPORT_REQUEST_RESPONSE_URL;
 
-const protobuf = REPORT_TYPE === 'proto' ? require('protobufjs') : null;
+const protobufLoad = REPORT_TYPE === 'proto' ? require('protobufjs').load : null;
```

Comment:
Does this prevent us from loading other parts of the library or something 🤔

Reply:
No, initially I retrieved `load` directly, as I wanted to _promisify_ it, but then in doc I've found that it returns a promise if callback is not passed (so no need for promisification), Still, I kept the above change.

No real difference here

Reply:
Ahh okay no worries just wanted to make sure I wasn't missing out on some node magic here 😎